tox.ini

```diff
@@ -35,5 +35,6 @@ commands =
      ;  bash ./continuous_integration/build_test_ext.sh
      python -c "import struct; print('platform: %d' % (8 * struct.calcsize('P')))"
      python continuous_integration/install_coverage_subprocess_pth.py
-     py.test {posargs:-vl --maxfail=5 --timeout=60}
+     py.test {posargs:-vl--maxfail=5 --timeout=60}
      coverage combine --append
+     coverage xml -i
```

Comment:
Why? Is it used by codecov? Maybe add a comment to explain.

Reply:
I *think* this is some leftover debug code. Let me check.